src/otel/OTelGroupCallMembership.ts

```diff
@@ -128,6 +128,7 @@ export class OTelGroupCallMembership {
     if (!ElementCallOpenTelemetry.instance) return;
     if (this.callMembershipSpan !== undefined) {
       logger.warn("Call membership span is already started");
+      this.callMembershipSpan.addEvent("matrix.doubleJoin");
```

Comment:
Perhaps it would be better to track this sort of thing by instrumenting `GroupCall.enter()` and `GroupCall.leave()` directly